### PR TITLE
Made MonkeyPatchManager's  file path configurable.

### PR DESCRIPTION
**Pull request by luckydonald**
For #239.
Made same changes in both `2.x` and `3.x` folders.

Note, as I do the edits on a different computer, they are not tested 😅.

### CODE REVIEW

**Comment by luckydonald** [comment]
The coverage decrease is due to more lines in an already untested function resulting in more untested lines.

**Comment by ytetsuro** [comment]
I guess maybe should you write test code.
and attachment that PR URL.

see: https://github.com/kenjis/ci-phpunit-test/blob/master/CONTRIBUTING.md#test-code
> Please write test code for your code and send Pull Request to the repository above. I don't accept your Pull Request if the code coverage is down.

**Comment by luckydonald** [comment]
It has no coverage before, so that kinda is great for a separate PR.

**Comment by ytetsuro** [comment]
I've written a test code.
There is a bug in your code.

https://github.com/kenjis/ci-app-for-ci-phpunit-test/pull/16

File: application/tests/_ci_phpunit_test/patcher/2.x/MonkeyPatchManager.php [hunk 1]
```diff
@@ -70,7 +74,11 @@ protected static function setDebug(array $config)
 		{
 			self::$debug = $config['debug'];
 		}
-		if (self::$debug)
+		if (isset($config['log_file']))
+		{
+			self::$debug = $config['log_file'];
```
Comment (ytetsuro) [inline application/tests/_ci_phpunit_test/patcher/2.x/MonkeyPatchManager.php]: The variable name is incorrect.
`self::$debug` -> `self::$log_file`

File: application/tests/_ci_phpunit_test/patcher/3.x/MonkeyPatchManager.php [hunk 2]
```diff
@@ -70,7 +74,11 @@ protected static function setDebug(array $config)
 		{
 			self::$debug = $config['debug'];
 		}
-		if (self::$debug)
+		if (isset($config['log_file']))
+		{
+			self::$debug = $config['log_file'];
```
Comment (ytetsuro) [inline application/tests/_ci_phpunit_test/patcher/3.x/MonkeyPatchManager.php]: The variable name is incorrect.
`self::$debug` -> `self::$log_file`

**Comment by luckydonald** [comment]
Indeed.
Good that you did the testing...
😅

**Comment by kenjis** [comment]
Merged manually: https://github.com/kenjis/ci-phpunit-test/commit/4696a6387b87098452f7f69a10214644ada677e2

**Pull request closed without merging**